src/apps/opening-hours-editor/Dialog.tsx

```diff
@@ -0,0 +1,39 @@
+// The dialog element allready has keyboard support
```

Comment:
Would it be an idea to make it a global component?
It seems very generic and we could use it in other contexts/apps?
As far as I remember we do not have a dialog component already...